README.md

```diff
@@ -1 +1,72 @@
-# HelloMicroverse
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse 
+
+> This is my first project. Setting up a "Hello world" repository to practice essential git skills
+
+![screenshot](screenshot.png)
+
+Additional description about the project and its features.
+
+## Built With 
+
+- html
+- css 
+
+
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
+
+## Authors
+
+👤 **Reem**
+
+- GitHub: [@githubhandle](https://github.com/Reem-lab)
+- Twitter: [@twitterhandle](https://twitter.com/Rem79940127)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/reem-janina-ab74ab21a/)
+
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] In this case, your MIT license link shows an error link. I would suggest you add the MIT.md license to your files. You can find the MIT license in this link [MIT license link](https://github.com/microverseinc/readme-template/blob/master/MIT.md). Also do not forget to add your name too. 😄